hello-slack/prd.md

```
@@ -0,0 +1,301 @@
+⸻
+
+:bookmark: Next.js（Firebase App Hosting）
+
+フロント側は、Next.jsアプリをFirebase Hostingで提供し、将来的にユーザーが操作するUIや認証を提供する。
+
+npx create-next-app frontend
+cd frontend
+firebase init hosting
+firebase deploy
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)